src/ui/dashboard.js

```diff
@@ -354,15 +289,8 @@ export class DashboardScene extends Phaser.Scene {
             });
 
         // Add scrollable window for clan members
-        if (!Object.keys(this.clan.scrollWindow).length) {
-            this.clan.scrollWindow = new ScrollWindow("clans");
-            this.scene.add(
-                "scroll-window",
-                this.clan.scrollWindow,
-                true,
-                this.characterData
-            );
-        }
+        this.clan.scrollWindow = new ScrollWindow("clans");
+        this.scene.add("scroll-window", this.clan.scrollWindow, true, this.characterData);
```

Comment:
I don't think this scroll window works - not sure if you've fixed in the future or not

Reply:
I _think_ I made it better but it's still finicky sometimes